utils/src/main/java/com/cloud/utils/UuidUtils.java

```diff
@@ -28,6 +28,10 @@ public final static String first(String uuid) {
         return uuid.substring(0, uuid.indexOf('-'));
     }
 
+    public static boolean isUuid(String data) {
```

Comment:
Why create this method?

Reply:
From my point of view, using validate in the name of a method gives the impression that if the validation fails, an exception will be thrown. As in this case only true or false is returned, I decided to create a method whose name, again from my point of view, makes more sense.

Reply:
Was that your point @JoaoJandre ?
I think the question was to use an existing `validateUUID` method instead of creating this new one, @SadiJr.

Reply:
I see, but then why not simply refactor the old method's name?

Reply:
depends on what does `regex.matches()` do on no match; if it return false yes than renaming it makes sense. If it throws an exeption than validate... is the right name for it.

Reply:
As far as I know, `regex.matches()` does not throw any exceptions, it returns false when the String doesn't match the regex.

Reply:
ok, so then it makes sense not to xall the method validate...